### DISCUSSION TITLE
How to create a data table with legend keys for a chart?

### DISCUSSION THREAD

**Discussion by sstylianides** [qax-os/excelize#1580] [Q&A] [answered]
Hi,

I'm able to create a stacked column chart with the proper names, categories, and values just fine but I want to add a data table with legend keys on the X axis. Doing it manually on Excel for Mac, I would: click on the already created chart > Chart Design > Add Chart Element > Data Table > With Legend Keys

How can I do that with excelize?

For reference attached is what I'd like the final output to be.

![data_table_w_legend](https://github.com/qax-os/excelize/assets/25205010/315765e3-34ab-4191-b5a7-58e88776dd4e)

**Comment by xuri** [comment, marked as answer]
Thanks for your feedback. This library doesn't support this feature currently, I'll consider to add support for this.

**Reply by sstylianides** [reply]
Thank you. I appreciate your response.
